### ISSUE TITLE
Seeing Error: Cannot find module '../package.json' when importing @traceloop/node-server-sdk

### ISSUE THREAD

**Issue by azhong-git** [traceloop/openllmetry-js#241]
Hi I saw the following error when importing @traceloop/node-server-sdk in a nodejs playground in a backend project. (I was using llama index and following the official example to use OpenLLMetry)

```
Error: Cannot find module '../package.json'
Require stack:
- node_modules/@traceloop/instrumentation-chromadb/dist/src/instrumentation.js
- node_modules/@traceloop/instrumentation-chromadb/dist/src/index.js
- node_modules/@traceloop/node-server-sdk/dist/index.js
- playground/test-traceloop.cjs
    at Module._resolveFilename (node:internal/modules/cjs/loader:1048:15)
    at Module._load (node:internal/modules/cjs/loader:901:27)
    at Module.require (node:internal/modules/cjs/loader:1115:19)
    at require (node:internal/modules/helpers:130:18)
    at Object.<anonymous> (node_modules/@traceloop/instrumentation-chromadb/dist/src/instrumentation.js:31:24)
    at Module._compile (node:internal/modules/cjs/loader:1233:14)
    at Module._extensions..js (node:internal/modules/cjs/loader:1287:10)
    at Module.load (node:internal/modules/cjs/loader:1091:32)
    at Module._load (node:internal/modules/cjs/loader:938:12)
    at Module.require (node:internal/modules/cjs/loader:1115:19) {
  code: 'MODULE_NOT_FOUND',
  requireStack: [
    'node_modules/@traceloop/instrumentation-chromadb/dist/src/instrumentation.js',
    'node_modules/@traceloop/instrumentation-chromadb/dist/src/index.js',
    'node_modules/@traceloop/node-server-sdk/dist/index.js',
    'playground/test-traceloop.cjs'
  ]
}
```

Here is the code I use:
```typescript
import * as traceloop from "@traceloop/node-server-sdk";

traceloop.initialize({
  disableBatch: true,
  apiKey:
    "some key",
});

```

The tsconfig
```
{
  "compilerOptions": {
    "target": "ES2021",
    "module": "commonjs",
    "lib": ["es2021", "dom"],
    ...
   },
  "exclude": ["node_modules", "cdk.out"]
}

```

This prevents me from using OpenLLMetry in backend to track LLM use in Llama index. Any help is appreciated

**Comment by nirga** [comment]
Thanks for reporting this @azhong-git! I found the problem and I'm issuing a fix now.

**Comment by nirga** [comment]
@azhong-git this is fixed with v0.8.1. Let me know if that works for you!
